tools/build/Makefile-Moar.in

```diff
@@ -38,19 +38,31 @@ P5QREGEX_MOAR     = NQPP5QRegex.moarvm
 NQP_MOAR     = nqp.moarvm
 M_NQP_SOURCES= src/vm/moar/NQP/Ops.nqp $(COMMON_NQP_SOURCES)
 
-M_STAGE1_OUTPUT = $(M_STAGE1_DIR)/$(NQP_MO_MOAR) $(M_STAGE1_DIR)/$(MODULE_LOADER_MOAR) \
-                $(M_STAGE1_DIR)/$(CORE_SETTING_MOAR) $(M_STAGE1_DIR)/$(QASTNODE_MOAR) \
-                $(M_STAGE1_DIR)/$(QREGEX_MOAR) $(M_STAGE1_DIR)/$(HLL_MOAR) \
-                $(M_STAGE1_DIR)/$(MASTOPS_MOAR) $(M_STAGE1_DIR)/$(MASTNODES_MOAR) \
-                $(M_STAGE1_DIR)/$(QAST_MOAR) $(M_STAGE1_DIR)/$(P6QREGEX_MOAR) \
-                $(M_STAGE1_DIR)/$(NQP_MOAR)
-
-M_STAGE2_OUTPUT = $(M_STAGE2_DIR)/$(NQP_MO_MOAR) $(M_STAGE2_DIR)/$(MODULE_LOADER_MOAR) \
-                $(M_STAGE2_DIR)/$(CORE_SETTING_MOAR) $(M_STAGE2_DIR)/$(QASTNODE_MOAR) \
-                $(M_STAGE2_DIR)/$(QREGEX_MOAR) $(M_STAGE2_DIR)/$(HLL_MOAR) \
-                $(M_STAGE2_DIR)/$(MASTOPS_MOAR) $(M_STAGE2_DIR)/$(MASTNODES_MOAR) \
-                $(M_STAGE2_DIR)/$(QAST_MOAR) $(M_STAGE2_DIR)/$(P6QREGEX_MOAR) \
-                $(M_STAGE2_DIR)/$(NQP_MOAR)
+M_STAGE1_OUTPUT = \
+  $(M_STAGE1_DIR)/$(NQP_MO_MOAR) \
+  $(M_STAGE1_DIR)/$(MODULE_LOADER_MOAR) \
+  $(M_STAGE1_DIR)/$(CORE_SETTING_MOAR) \
+  $(M_STAGE1_DIR)/$(QASTNODE_MOAR) \
+  $(M_STAGE1_DIR)/$(QREGEX_MOAR) \
+  $(M_STAGE1_DIR)/$(HLL_MOAR) \
+  $(M_STAGE1_DIR)/$(MASTOPS_MOAR) \
+  $(M_STAGE1_DIR)/$(MASTNODES_MOAR) \
+  $(M_STAGE1_DIR)/$(QAST_MOAR) \
+  $(M_STAGE1_DIR)/$(P6QREGEX_MOAR) \
+  $(M_STAGE1_DIR)/$(NQP_MOAR)
```

Comment:
Maybe you should add a trailing backslash to the last line while you're at it, same as in Makefile-JVM.in?